include/nuttx/wireless/ioctl.h

```diff
@@ -57,16 +57,58 @@
 #define WLIOC_GETADDR       _WLCIOC(0x0004)  /* arg: Pointer to address value, format
                                               * of the address is driver specific */
 #define WLIOC_SETTXPOWER    _WLCIOC(0x0005)  /* arg: Pointer to int32_t, */
-                                             /* output power (in dBm) */
+                                             /* output power (in 0.01 dBm) */
```

Comment:
this is true only for this driver. All the other drivers use the old unit. Either all other usage of this command should be fixed or this comment should say that this value depends on the driver

Reply:
You're right, it was my intention to change this globally once we decided on a better LoRa framework, but I can patch the docs if that's necessary.